README.md

```diff
@@ -72,7 +72,11 @@ as ARM and MIPS. Making full-featured port to another architecture requires
 writing some assembly code for the exception handling and garbage collection.
 Alternatively, fallback implementation based on setjmp/longjmp can be used.
 
-To build (see section below for required dependencies):
+**To build:**
+
+First, build the required dependencies. (See section below.)
+
+Then build micropython:
```

Comment:
I don't think this part needs to be changed because the existing wording tells that there are required dependencies, and is nice and concise.  So please remove this change from the PR.